cookbook/https.rst

```diff
@@ -0,0 +1,35 @@
+HTTPS
+=====
+
+Sulu behave the same way as Symfony when generating urls.
+If the requested protocol is ``https`` it will use automatically ``https`` to generate its urls.
+
+If this does not happen its a hint that you are behind a load balancer or have a proxy between your request server.
```

Comment:
"If this does not work properly, that your setup includes some kind of proxy, e.g. a load balancer or a HTTP cache like Varnish."

Reply:
Yeah, that's right... But if at all it would be `it's a hint`. (its shows some kind of possession). But I think I would like it even better this way:

"If this does not work propertly, your setup probably includes some kind of proxy, e.g. a load balancer or a HTTP cache like Varnish."

Reply:
@danrot fixed!